src/DotNetWorker.Grpc/DotNetWorker.Grpc.csproj

```diff
@@ -55,6 +55,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\extensions\Worker.Extensions.Abstractions\src\Worker.Extensions.Abstractions.csproj" />
+    <ProjectReference Include="..\..\extensions\Worker.Extensions.Rpc\src\Worker.Extensions.Rpc.csproj" />
```

Comment:
hmm I am not entirely sold on having this reference. At least not yet, the RPC package is still preview and has not yet been published to nuget.

Even then, the goal of the Extensions.Rpc project is to assist other extensions with using custom RPC back to the host. If we do want DotnetWorker.Grpc to use this - which it could - it would be a larger effort to have Extensions.Rpc to have a public API surface that meets its needs better.

Reply:
Sounds good to me. @fabiocav let me know you have concerns because t[his comment ](https://github.com/Azure/azure-functions-dotnet-worker/pull/1897#discussion_r1325152768)of yours will not be addressed.

Reply:
Pushed an update to address this.